### PR TITLE
tests(events-v2) Add acceptance tests and percy for events modal

### PR DESCRIPTION
**Pull request by markstory**
With the modal approaching completion it would be good to have acceptance tests and percy snapshots to help keep it working in the future.

Refs SEN-731

### CODE REVIEW

File: tests/acceptance/test_organization_events_v2.py [hunk 1]
```diff
@@ -84,5 +90,75 @@ def test_errors(self, mock_now):
 
         with self.feature(FEATURE_NAME):
             self.browser.get(self.path + '?view=errors')
-            self.browser.wait_until_not('.loading-indicator')
-            self.browser.snapshot('events-v2 - errors')
+            self.wait_until_loaded()
+            self.browser.snapshot('events-v2 - error list')
```
Comment (lynnagara) [inline tests/acceptance/test_organization_events_v2.py]: Do we need this snapshot? It's pretty similar to `events-v2 - all events`

Comment (markstory) [reply tests/acceptance/test_organization_events_v2.py]: I could add more events so that there is more than one group.

Comment (lynnagara) [reply tests/acceptance/test_organization_events_v2.py]: Ah right, could we add extra events to to the `events-v2 - errors` snapshot instead? That snapshot is supposed to capture this tab

Comment (markstory) [reply tests/acceptance/test_organization_events_v2.py]: Sure.

**Comment by markstory** [comment]
@lynnagara Tests have been updated.

**Review by lynnagara — APPROVED** [review]

**Pull request merged**